src/Grpc/test/testassets/InteropClient/InteropClient.cs

```diff
@@ -179,7 +180,20 @@ private async Task<IChannelWrapper> HttpClientCreateChannel()
             return new GrpcChannelWrapper(channel);
         }
 
-        private bool IsHttpClient() => string.Equals(options.ClientType, "httpclient", StringComparison.OrdinalIgnoreCase);
+        // TODO(JamesNK): This type can be removed in the future when Grpc.Net.Client sets VersionPolicy automatically.
+        // https://github.com/grpc/grpc-dotnet/pull/987
+        private class VersionPolicyHandler : DelegatingHandler
```

Comment:
nit: Don't we put inner classes at the bottom of our files❔

Reply:
I agree, but this class is temporary. It will disappear in a few days.